doc/examples/nonlinear_from_rest/run_simul_check_from_python.py

```diff
@@ -113,95 +154,95 @@ def main(args):
     sim = Simul(params)
-    sim.output.write_snakemake_config(
-        custom_env_vars={"MPIEXEC_FLAGS": "--report-pid PID.txt"}
-    )
+    # sim.output.write_snakemake_config(
+    #     custom_env_vars={"MPIEXEC_FLAGS": "--report-pid PID.txt"}
+    # )
 
-    sim.make.exec("run", resources={"nproc": args.nb_mpi_procs})
+    sim.make.exec("run_fg", resources={"nproc": args.nb_mpi_procs})
     return params, sim
 
 
 if __name__ == "__main__":
     args = parser.parse_args()
     params, sim = main(args)
 
-    pid_file = sim.path_run / "PID.txt"
-
-    n = 0
-    while not pid_file.exists():
-        sleep(1)
-        n += 1
-        if n > 60:
-            raise RuntimeError(f"{pid_file} does not exist.")
-
-    with open(pid_file) as file:
-        pid = int(file.read().strip())
-
-    path_log_py = sim.path_run / f"log_py_{time_as_str()}.txt"
-
-    def print(*args, sep=" ", end="\n", **kwargs):
-        builtins.print(*args, **kwargs)
-        with open(path_log_py, "a") as file:
-            file.write(sep.join(str(arg) for arg in args) + end)
-
-    print(f"{pid = }")
-
-    def check_running():
-        """Check For the existence of a unix pid."""
-        try:
-            os.kill(pid, 0)
-        except OSError:
-            return False
-        else:
-            return True
-
-    while check_running():
-        sleep(10)
-        # TODO: make this call faster even for large .his file
-        # see https://github.com/exabl/snek5000/issues/108
-        # and https://github.com/exabl/snek5000/tree/faster-history-points-load
-        t0 = perf_counter()
-        coords, df = sim.output.history_points.load_1point(
-            index_point=5, key="temperature"
-        )
-        t_last = df.time.max()
-        print(
-            f"{time_as_str()}, {t_last = :.2f}: "
-            f"history_points loaded in {perf_counter() - t0:.2f} s"
-        )
-
-        if t_last < 800:
-            continue
-
-        temperature = df.temperature
-        times = df.time
-
-        duration_avg = 200.0
-
-        temp0_std = np.std(
-            temperature[
-                (t_last - 2 * duration_avg < times) & (times < t_last - duration_avg)
-            ]
-        )
-        temp1_std = np.std(
-            temperature[(t_last - duration_avg < times) & (times < t_last)]
-        )
-
-        print(f"  {abs(temp0_std - temp1_std) / temp0_std = :.3f}, {temp1_std = :.3g}")
-        if temp1_std < 1.0e-14:
-            print(
-                f"Steady state (stable) detected at t = {t_last}\n"
-                "Terminate simulation."
-            )
-
-            os.kill(pid, signal.SIGTERM)
-            break
-
-        if abs(temp0_std - temp1_std) / temp0_std < 0.2 and temp1_std > 0.0004:
-            print(
-                f"Saturation of the instability detected at t = {t_last}\n"
-                "Terminate simulation."
-            )
-
-            os.kill(pid, signal.SIGTERM)
-            break
+    # pid_file = sim.path_run / "PID.txt"
```

Comment:
Why do you comment these lines? It is nice to have them in this example script.

Reply:
ok.